### PR TITLE
Update to remove tslint from validator - Partially Closes #4696 

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR partially resolves #4696 

### How was it solved?

- Remove tslint from lisk-validator

### How was it tested?
- All the linting should work
- no source code changes

### CODE REVIEW

File: elements/lisk-validator/src/lisk_validator.ts [hunk 1]
```diff
@@ -31,12 +31,14 @@ class LiskValidator {
 		for (const formatName of Object.keys(formats)) {
 			this.validator.addFormat(
 				formatName,
+				// eslint-disable-next-line import/namespace
```
Comment (pablitovicente) [inline elements/lisk-validator/src/lisk_validator.ts:34]: isn't this rules for imports only?

Comment (shuse2) [reply elements/lisk-validator/src/lisk_validator.ts:34]: I think this is showing error because of `as keyof typeof` for typescript

Comment (pablitovicente) [reply elements/lisk-validator/src/lisk_validator.ts:34]: If possible it would be nice to disable that one then. Approved anyway.

Comment (shuse2) [reply elements/lisk-validator/src/lisk_validator.ts:34]: I think this is the only case so far, so i would like to keep it for now.
If it continues to report wrong value, we can disable them

**Pull request merged**